CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+Substrate Sidecar API is an Open Source project that welcomes new issues and PRs.
+
+If you’re new to open source or interested in contributing, this guide: [How to Contribute to Open Source](https://opensource.guide/how-to-contribute/) is particularly helpful.
+
+## Contributing Code
+To contribute to this repo, it’s recommended to create an issue first; this allows you to gather feedback before writing any code. In the issues tab, you’ll be able to select from a template to create an issue. 
+
+To create a Pull Request after you’ve created an issue, you’ll need to head over to the Pull Request tab. This will be where all the PRs will be located, and where reviewers will be able to review your changes made. 
```

Comment:
```suggestion
To create a Pull Request after you’ve created an Issue, you’ll need to head over to the Pull Request tab. This will be where all the PRs will be located, and where reviewers will be able to review your changes made. 
```